tests/functional/test_pkgutil.py

```diff
@@ -43,9 +45,9 @@ def _read_results_file(filename):
 @pytest.mark.parametrize(
     'package',
     [
-        'altgraph',  # pure python package
-        'psutil',  # package with extensions
-        'psutil.tests',  # sub-package
+        'json',  # pure python package (stdlib)
+        'xml.dom',  # sub-package (stdlib)
+        'psutil',  # package with extensions (3rd party)
     ]
```

Comment:
Is there also a _single file package_ permutation? e.g. mistune (one `.py` file) or ujson (one extension module)?

Reply:
So a top-level module? No, because we are testing iterating over modules with `pkgutil.iter_modules˙.

If I remember correctly, the idea was to have a pure-python package to see if we iterate correctly over modules in PYZ, a sub-package just in case there's any sub-path related issues, and a package with extensions (because then you have a mixture of modules in PYZ and binary extensions as external files).

Reply:
And I don't think the test script would work for modules due to lack of `__path__` attribute:

https://github.com/pyinstaller/pyinstaller/blob/6fd33d426654c3a4263cfe3b94e9da3f33ef3785/tests/functional/scripts/pyi_pkgutil_iter_modules.py#L52-L57